app/client/views/admin/users/users.html

```diff
@@ -87,6 +89,53 @@ <h3> {{section.title}} </h3>
         </div>
       </div>
 
+      <!-- Hardware Modal -->
+      <div class="ui long hardware modal">
+        <i class="close icon"></i>
+        <div class="header">
+          Hardware for {{selectedUser.profile.name ? selectedUser.profile.name : selectedUser.email}}
+        </div>
+        <div class="content">
+          <div class="ui form">
+            <fieldset>
+              <div class="fields" ng-repeat="item in selectedUser.hardware track by $index">
+                  <div class="nine wide field" >
+                    <select ng-model="item.name">
+                      <option value="">Select Hardware</option>
+                      <option ng-repeat="h in hardware" ng-if="!isCheckedOut(h, item)" value="{{h}}">{{h}}</option>
+                    </select>
+                  </div>
+                  <div class="two wide field">
+                    <input type="number" value="1" ng-model="item.amount"/>
```

Comment:
min/max values for minor data integrity? Given that this is an admin thing, I don't think we need to check server/js side but maybe something to think about.